Hyperledger_Fabric_Technical_Charter.md

```diff
@@ -1,116 +1,163 @@
 
-6. **General Rules and Operations.** 
+        3.  Each such **Sub-Project's** Maintainer group shall determine its collective approval or disapproval of the proposal via its own internal voting process (a simple majority vote of its active members).
 
-   1. The Project will:
+        4.  Each **Sub-Project's** Maintainer group then signals its single collective decision (approve/disapprove) to the TSC.
 
-      1. engage in the work of the Project in a professional manner consistent with maintaining a cohesive community, while also maintaining the goodwill and esteem of LF Projects, Joint Development Foundation and other partner organizations in the open source community; and
+        5.  The TSC tallies these group decisions. If the required simple majority of group approvals is met, the proposal is considered adopted by the ecosystem.
 
-      2. respect the rights of all trademark owners, including any branding and trademark usage guidelines.
+        6.  Matters requiring this vote include, but are not limited to:
 
-7. **Intellectual Property Policy**
+            1.  Formal acceptance and recognition of a new **Sub-Project** into the ecosystem.
 
-   1. Collaborators acknowledge that the copyright in all new contributions will be retained by the copyright holder as independent works of authorship and that no contributor or copyright holder will be required to assign copyrights to the Project. 
+            2.  Archiving or sunsetting an existing recognized **Sub-Project**.
 
-   2. Except as described in Section 7.c., all contributions to the Project are subject to the following: 
+            3.  Making significant changes to any TSC-defined core principles or guidelines.
 
-      1. All new inbound code contributions to the Project must be made using Apache License, Version 2.0 available at http://www.apache.org/licenses/LICENSE-2.0  (the “Project License”). 
+            4.  Adopting or significantly changing other ecosystem-wide policies.
 
-      2. All new inbound code contributions must also be accompanied by a Developer Certificate of Origin ([http://developercertificate.org](http://developercertificate.org)) sign-off in the source code system that is submitted through a TSC-approved contribution process which will bind the authorized contributor and, if not self-employed, their employer to the applicable license;
+    4.  Amendments to this Charter: Amendments to this Technical Charter (as per Section 10) shall require a **supermajority** vote of the distinct Sub-Project Maintainer groups.
 
-      3. All outbound code will be made available under the Project License.
+    5.  Role of the TSC: The TSC is responsible for facilitating this voting process, formally defining criteria for a **Sub-Project** to be considered "active" and thus eligible to participate, setting the precise majority threshold, ratifying the outcomes of these votes, and documenting all such decisions.
```

Comment:
We have to start with an initial threshold; if this is ratified we have to definition of super-majority

Reply:
Added a separate Definition section.